setup.py

```diff
@@ -12,7 +12,7 @@
     license='Apache License Version 2.0',
     author='JasperCraeghs',
     author_email='jce@melexis.com',
-    description='Python script for converting xUnit/JUnit XML format to reStructuredText format',
+    description='Python script for converting xUnit/JUnit XML format to RST format with traceable items',
```

Comment:
I liked reStructuredText here... it will be better with hits, so maybe change into: `reStructuredText (.rst)`

Reply:
Changed in b87b6bf9b6a8a8ac86a828c7dd9a45adf31c0052